utils/build-all.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/sh
 # kos-ports ##version##
 #
-# utils/build-all.mk
+# utils/build-all.sh
 # Copyright (C) 2015 Lawrence Sebald
+# Copyright (C) 2024 Andy Barajas
 #
 
 cd ${KOS_PORTS}
+errors=""
+error_count=0
 
 for _dir in ${KOS_PORTS}/* ; do
```

Comment:
```suggestion
for _dir in $(realpath ${KOS_PORTS})/* ; do
```

If you don't mind, I'd like to suggest here to use `realpath` (POSIX compliant) to make the output look better. For example, before:

```
/opt/toolchains/dc/kos/../kos-ports/cglm: Build failed with return code 2
/opt/toolchains/dc/kos/../kos-ports/libchipmunk: Build failed with return code 2
/opt/toolchains/dc/kos/../kos-ports/libcmark: Build failed with return code 2
/opt/toolchains/dc/kos/../kos-ports/mruby: Build failed with return code 2
```

After:

```
/opt/toolchains/dc/kos-ports/cglm: Build failed with return code 2
/opt/toolchains/dc/kos-ports/libchipmunk: Build failed with return code 2
/opt/toolchains/dc/kos-ports/libcmark: Build failed with return code 2
/opt/toolchains/dc/kos-ports/mruby: Build failed with return code 2
```

This also makes things prettier throughout the rest of the script's text, too.